### PR TITLE
Added schedule and Abstracts to workshop page

### PR DESCRIPTION
**Pull request by Ermine516**

### CODE REVIEW

File: _pages/WG5/Prague23/meeting.md [hunk 1]
```diff
@@ -33,23 +37,56 @@ Both workshops solicit contributed talks backed up by an extended abstract of up
 
 The travel and accommodation of a number of participants (approximately 15) will be supported by the [Cost Action CA20111 - European Research Network on Formal Proofs](https://europroofnet.github.io/).
 If you want to be funded, please
-check the [eligibility](https://europroofnet.github.io/eligibility) and [reimbursement](https://europroofnet.github.io/reimbursement-rules/) rules to know whether you can be funded.
+Check the [eligibility](https://europroofnet.github.io/eligibility) and [reimbursement](https://europroofnet.github.io/reimbursement-rules/) rules to know whether you can be funded.
```
Comment (fblanqui) [inline _pages/WG5/Prague23/meeting.md:40]: Shouldn't this line start with lowercase?

**Review by fblanqui — CHANGES_REQUESTED** [review]
Hi. Thank you for your PR. I noticed a few details to fix. Moreover, could you please move the pdf files from the asset directory to the _pages/WG5/Prague23/ directory? I prefer to keep documents locally.

**Comment by Ermine516** [comment]
Concerning assets. I had a problem with putting the PDFs in the same directory.  Is some trick to this? I followed how the other pages did it but it did not work

**Comment by Ermine516** [comment]
I made all the changes, and found the error when linking to files.

**Comment by fblanqui** [comment]
Thank you!

**Pull request merged**